runner/src/server/plugins/initialiseSession/types.ts

```diff
@@ -6,6 +6,7 @@ export type InitialiseSessionOptions = {
   callbackUrl: string;
   redirectPath?: string;
   message?: string;
+  redirectToUrl?: string;
```

Comment:
would `returnUrl` make more sense if this is the URL the user goes back to after completing the form?